### ISSUE TITLE
Overdue query seems that it doesn't retrieve tasks

### ISSUE THREAD

**Issue by HoracioDos** [Garee/pytodoist#8]
Hello:
I have the following code and the overdue query seems that it doesn't retrieve tasks. I created a dummy overdue task.

from pytodoist import todoist
user = todoist.login('user@domain.com', 'password')
tasks = user.search_tasks(todoist.Query.OVERDUE)
for tasks in tasks:
     print task.content

When I replace the query for TODAY it works perfect!
tasks = user.search_tasks(todoist.Query.TODAY)

Thank you in advance.
This library is amazing!

![overdue](https://cloud.githubusercontent.com/assets/4549674/7101098/4fcf61ce-e01c-11e4-91a9-fdd848dd562a.png)

**Comment by Garee** [comment]
Hello,

Unfortunately I'm unable to replicate the issue. Is there any other information that you could provide to help me debug? Are you using the latest version of pytodoist 1.0.0?

![capture](https://cloud.githubusercontent.com/assets/861811/7101140/2904eb48-e041-11e4-9a01-2bd0576f79d7.PNG)

```python
>>> from pytodoist import todoist
>>> user = todoist.login(...)
>>> tasks = user.search_tasks(todoist.Query.OVERDUE)
>>> for task in tasks:
...     print(task.content)
...
Overdue Task
>>>
```

**Comment by HoracioDos** [comment]
Hello: 
Yes!, I'm using the latest version of pytodoist 1.0.0
pip freeze | grep pytodoist
pytodoist==1.0.0
I copied the exact same code that you wrote and I get the same result. No overdue tasks.
The attached image file shows that I have an overdue task.

What kind of information do you need to help you?

Thank you very much.

**Comment by HoracioDos** [comment]
I forgot to tell you that I tested the code in a linux mint 17.1 box and in a raspberry pi with same results.

**Comment by HoracioDos** [comment]
Hi:
I found something else today . If tasks are only 1 day old, they are not retrieved, but if tasks are older than 2 days, the query works OK.
Thanks.

**Comment by Garee** [comment]
I just released version 2.0 which implements support for the new Todoist API v6. Can you upgrade and see if that solves your issue?

**Comment by HoracioDos** [comment]
Cool!! I'll test it ASAP.

Thank you very much!
H

**Comment by HoracioDos** [comment]
Hi Garee: 

I installed version 2.0 today and the results are the same. If tasks are only 1 day old, they are not retrieved, but if tasks are older than 2 days, the query works OK. 

I also tested the API directly and the output is always the same.

import todoist

api = todoist.TodoistAPI()
request = api.login('name@domain.com', 'pwd')
api_token = request['api_token']
api = todoist.TodoistAPI(api_token)
request = api.query(['overdue'])

I think it's not the wrapper, perhaps there is something wrong with my account.
I'm going to check this behaviour with customer support

Thank you for your time.
